include/picongpu/particles/atomicPhysics/ionizationPotentialDepression/kernel/CalculateIPDInput.kernel

```diff
@@ -135,7 +136,10 @@ namespace picongpu::particles::atomicPhysics::ionizationPotentialDepression::ker
                 {
                     /* unit: (unitless * weight / typicalNumParticlesPerMacroParticle)
                      *  / (unitless * weight / typicalNumParticlesPerMacroParticle) = unitless */
-                    zStarBox(superCellFieldIdx) = sumChargeNumberSquared / sumChargeNumber;
+                    zStarBox(superCellFieldIdx)
+                        = (math::abs(sumChargeNumber) < std::numeric_limits<float_X>::epsilon())
+                              ? sumChargeNumberSquared / sumChargeNumber
+                              : 0._X;
```

Comment:
shouldn't that be interchanged?

Reply:
if `math::abs(sumChargeNumber)` you want to have zero, right? 
That's not the case right now